README_SETUP.md

```diff
@@ -95,6 +95,38 @@ You also need `easy_install` to install additional packages.
 </p></details>
 
 
+
+### Required Software - DotNet 9
+
+(optional)
```

Comment:
I'll keep this optional for now at least.
Currently, we only reliably support macOS, due to the state of DotNet 9